### ISSUE TITLE
<a href="#anchor"> does not link to <a name="anchor> in same page

### ISSUE THREAD

**Issue by marlonrichert** [vaadin/flow#1215]
Normally, when I create `<a href="#anchor">` in an HTML page, this will cause there to be link pointing to `<a name="anchor">` in that same page. So, for example, if you're on https://localhost:8080/my-page, then clicking the link will take you to https://localhost:8080/my-page/#anchor.

However, in ~Hummingbird~ (now known as Flow), when you create `<a href="#anchor">` on https://localhost:8080/my-page, clicking the link will take you to the root of the site instead, to https://localhost:8080/#anchor. This is not expected behavior.

Please make it behave like on a normal site.

**Comment by samie** [comment]
I feel that mixing Flow navigation in here is overkill. Developers are looking a quick way to let users navigate within the current page/view.


Code like this should just work as it works in browsers (without any Flow navigation/events happening): 

```
Link link = new Link("View feedback","#feedback");
myView.add(link); 
```

**Comment by pleku** [comment]
Yes, it should be easy to create inside-page hash-based links from that just work, either in a template or from Java.

No timeline for this enhancement currently though, would require some +1s

**Comment by gianluca-valentini** [comment]
I follow the issue as it is creating problem to our application

**Comment by vaadin-miki** [comment]
Following the issue as well, as it renders my current project basically unusable.

**Comment by bsutton** [comment]
Gentlefolk,
I'm current wedged by this bug.

Its marked as Done/Pending Release (as of May 2021) do we known when this is going to be released and/or the details of what the solution was?

I think the issue [#822](https://github.com/vaadin/router/issues/822) that I raised relates t this.
